observermedia/eight-day-week
# Tested WP 6.3 Compatibility and Fixed User Screen Bug

### Description of the Change
This PR includes the bumped up version for Tested upto to 6.3 and fixes the bug that was happening in the users screen when trying to change the print roles.
Closes #123 #124 

### How to test the Change
As admin > Go to Users list
Select a user other than the admin role
Select "Change print role to" > Eight-day week
Click on Change button and it should update the user without any errors.

### Changelog Entry

> Changed - Tested upto 6.3
> Fixed - User screen error message when updating print roles


### Credits
@zamanq @jeffpaul 


### Checklist:
